jenkins-brain.planx-pla.net/manifests/hatchery/hatchery.json

```
@@ -63,11 +63,10 @@
       "lifecycle-post-start": [
         "/bin/sh",
         "-c",
-        "export IAM=`whoami`; rm -rf /home/$IAM/pd/dockerHome; rm -rf /home/$IAM/pd/lost+found; ln -s /data /home/$IAM/pd/; true"
+        "export IAM=`whoami`; rm -rf /home/$IAM/pd/lost+found; true"
```

```suggestion
        "export IAM=`whoami`; rm -rf /home/$IAM/pd/dockerHome; rm -rf /home/$IAM/pd/lost+found; ln -s /data /home/$IAM/; true"
```
I think if you remove the /data link, then export to workspace file mopunt will fail